changelog/17.feature.rst

```diff
@@ -0,0 +1 @@
+Added a new function (`fourier_normalizing_radial_gradient_filter`) to normalize the radial brightness gradient using a fourier approximation.
```

Comment:
```suggestion
Added a new function (`sunkit_image.offlimb_enhance.fourier_normalizing_radial_gradient_filter`) to normalize the radial brightness gradient using a fourier approximation.
```

These imports are getting way too long.

Reply:
Should I change the function name to FNRGF?? It will shorten the imports

Reply:
Sure, try that and see how it goes. I also think the file itself needs renaming. I am not a fan of offlimb_enchance.

Reply:
Yes me too. I feels a little odd.

Reply:
Why not only ```offlimb```?? For the algorithms affecting the surface we can use the file name `surface`

Reply:
Well what about ones that affect both the surface and the limb?

Reply:
We could call it enhance or contrast?

Reply:
Enhance would be much better for algorithms affecting both.

Reply:
That would work.